### PR TITLE
README: add link to tldr.finzzz.net

### PR DESCRIPTION
**Pull request by finzzz**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit)

Fixes: #5287

### CODE REVIEW

**Comment by bl-ue** [comment]
I'm sorry @finzzz I meant issue **description**, not title.  I update it.

**Comment by bl-ue** [comment]
@user the CLA isn't working because the email address that you used for the commit (`git@finzzz.net`) isn't added to your GitHub account.  To fix this, you can either:
1. Change the email you used for the commit† to [one of the emails attached to your GitHub account](https://github.com/settings/emails),

2. Or, preferably (and much easier), [add the email you used for this commit (`git@finzzz.net`) to your GitHub account](https://github.com/settings/emails).

†To change the email address of the last commit, run this command:
```
git commit --amend --author="Vincent Carlos <email addres that you want to use instead>"
```

**Comment by finzzz** [comment]
okay, can you try again?

**Comment by bl-ue** [comment]
Okay, the commit looks good.  Now just [sign the CLA](https://cla-assistant.io/tldr-pages/tldr?pullRequest=5289).

**Review by bl-ue — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ both for the command line and for other platforms:
 - [Visual Studio Code extension](https://github.com/bmuskalla/vscode-tldr) available on [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=bmuskalla.vscode-tldr)
 - Web clients:
     - [tldr.jsx](https://github.com/ostera/tldr.jsx): http://tldr.ostera.io/
+    - [tldr.finzzz.net](https://git.finzzz.net/tldr/): https://tldr.finzzz.net/
```
Comment (navarroaxel) [inline README.md:144]:
```suggestion
    - [tldr.finzzz.net](https://git.finzzz.net/tldr/): https://tldr.finzzz.net
```

Comment (bl-ue) [reply README.md:144]: also the above link? tldr.ostera.io has a `/` at the end...

Comment (navarroaxel) [reply README.md:144]: Yeah, we can remove the that `/` too.

File: README.md [hunk 2]
```diff
@@ -141,6 +141,7 @@ both for the command line and for other platforms:
 - [Visual Studio Code extension](https://github.com/bmuskalla/vscode-tldr) available on [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=bmuskalla.vscode-tldr)
 - Web clients:
     - [tldr.jsx](https://github.com/ostera/tldr.jsx): http://tldr.ostera.io/
```
Comment (bl-ue) [inline README.md:143]: While you're here...
```suggestion
    - [tldr.jsx](https://github.com/ostera/tldr.jsx): http://tldr.ostera.io
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by sbrl — APPROVED** [review]
Awesome, thanks @finzzz!

The slashes are really just a neatness thing. We can either resolve that here, or open another quick PR to fix it. Either way, I don't think it's blocking this PR from being merged.

**Review by bl-ue — APPROVED** [review]
🚀🚀🚀

**Pull request merged**